wmn-data.json

```diff
@@ -6602,6 +6602,16 @@
         "m_code" : 404,
         "known" : ["carlito", "taylor"],
         "cat" : "social"
+      },
+      {
+        "name" : "Statuspage",
+        "uri_check" : "https://{account}.statuspage.io/api/v2/status.json",

```

Comment:
Since this check uses the API, please add a `"uri_pretty" : "https://{account}.statuspage.io/",` on the line following it so that people get redirected to the human-readable profile.